packages/node_modules/@webex/plugin-meetings/src/peer-connection-manager/index.js

```diff
@@ -323,14 +323,9 @@ pc.createOffer = (peerConnection, {meetingId, remoteQualityLevel}) => {
 
     return peerConnection
       .createOffer()
-      .then((description) => {
-        // bug https://bugs.chromium.org/p/chromium/issues/detail?id=1020642
-        // chrome currently generates RTX line irrespective of weither the server side supports it
-        // we are removing apt as well because its associated with rtx line
-        description.sdp = description.sdp.replace(/\r\na=rtpmap:\d+ rtx\/\d+/g, '');
-        description.sdp = description.sdp.replace(/\r\na=fmtp:\d+ apt=\d+/g, '');
-        peerConnection.setLocalDescription(description)
-      })
+      .then((description) => 
```

Comment:
i love when we can remove sdp munging :)

Reply:
But i feel that the issues with the RTX is still there in linus waiting for test to check it